numpy/fft/_helper.py

```diff
@@ -121,14 +121,13 @@ def ifftshift(x, axes=None):
 
     return roll(x, shift, axes)
 
-
```

Comment:
I don't understand where this change comes from?  (Shouldn't the linter even dislike it.)

The double space after `.` below doesn't matter, but is not really incorrect, just a style choice.
(Any good text editing program will make the space after a period longer, it depends on where you learned things whether or not you prefer two spaces to indicate this.)